model_analyzer/config/generate/neighborhood.py

```diff
@@ -273,13 +363,39 @@ def _compile_neighborhood_measurements(
         measurements = []
         for coordinate in visited_coordinates:
             measurement = self._coordinate_data.get_measurement(coordinate)
-            vectors.append(coordinate - self._home_coordinate)
-            measurements.append(measurement)
+            if measurement:
```

Comment:
We are no longer considering NONE measurements when making decisions?

Reply:
We don't. Previously when we were setting the None measurement weights to zeros, it didn't affect the step vector directions anyway (since adding zero vector does not change the step vector). Only thing that gets affected was its scale since the denominator will be slightly larger if we considered them. However, the fact that None measurement affects the step vector seemed a bit strange, so we simply just ignore them.